src/clr-angular/utils/_variables.clarity.scss

```diff
@@ -137,6 +137,3 @@
 
 // Accordion
 @import '../accordion/variables.accordion';
-
-// Typography
-@import '../typography/variables.typography';
```

Comment:
What is going on here?

Reply:
Duplicate import, was imported already at the top of the file.